forecasters/armadillo/DESCRIPTION

```diff
@@ -0,0 +1,27 @@
+Package: armadillo
+Title: Armadillo forecaster (Larry and Valerie's mobility model)
+Version: 0.0.0.9000
+Authors@R: 
+    person(given = "First",
+           family = "Last",
+           role = c("aut", "cre"),
+           email = "first.last@example.com",
+           comment = c(ORCID = "YOUR-ORCID-ID"))
```

Comment:
@Shitong-Wei you should list at least yourself. But probably also the rest of the team.

Reply:
The others will be done later.

Reply:
The authors and the license are added.